src/Aspire.Hosting/DistributedApplicationBuilder.cs

```diff
@@ -248,6 +248,11 @@ public DistributedApplicationBuilder(DistributedApplicationOptions options)
                 _innerBuilder.Services.TryAddEnumerable(ServiceDescriptor.Singleton<IValidateOptions<DashboardOptions>, ValidateDashboardOptions>());
             }
 
+            if (options.EnableResourceLogging)
+            {
+                _innerBuilder.Services.AddHostedService<ResourceLoggerForwarderService>();
```

Comment:
Maybe add a comment that this is explicitly being added before `// DCP stuff`, so we can try to avoid regressing it.

Reply:
Is there a test we can write to ensure it doesn't regress?

Reply:
Based on the failing `Aspire.Hosting.Containers.Tests.WithDockerfileTests.ContainerBuildLogsAreStreamedToAppHost` - maybe something similar could be a test.